tests/CMakeLists.txt

```
@@ -153,11 +153,10 @@ if(GEN_FILES)
             ${TF_PSA_CRYPTO_FRAMEWORK_DIR}/scripts/mbedtls_framework/psa_test_case.py
             ${TF_PSA_CRYPTO_FRAMEWORK_DIR}/scripts/mbedtls_framework/test_case.py
             ${TF_PSA_CRYPTO_FRAMEWORK_DIR}/scripts/mbedtls_framework/test_data_generation.py
-            ${CMAKE_CURRENT_SOURCE_DIR}/../include/psa/crypto_config.h
-            ${CMAKE_CURRENT_SOURCE_DIR}/../include/psa/crypto_values.h
-            ${CMAKE_CURRENT_SOURCE_DIR}/../include/psa/crypto_extra.h
+            ${TF_PSA_CRYPTO_crypto_config_path}
```

The generated file is independent of the configuration, so the python code should read to the source file, not the modified file in the build tree. And as far as I can tell, they do (they look for files in `include` relative to the source tree, without using different ways to look up e.g. `crypto_values.h` vs `crypto_config.h`).

```suggestion
            ${PROJECT_SOURCE_DIR}/include/psa/crypto_config.h
```

---

Yes good point, thanks.